polkadot/runtime/parachains/src/configuration/migration/v11.rs

```diff
@@ -18,16 +18,125 @@
 
 use crate::configuration::{self, Config, Pallet};
 use frame_support::{
-	migrations::VersionedMigration, pallet_prelude::*, traits::Defensive, weights::Weight,
+	migrations::VersionedMigration,
+	pallet_prelude::*,
+	traits::{Defensive, OnRuntimeUpgrade},
+	weights::Weight,
 };
 use frame_system::pallet_prelude::BlockNumberFor;
-use primitives::{vstaging::ApprovalVotingParams, SessionIndex};
+use primitives::{
+	vstaging::{ApprovalVotingParams, NodeFeatures},
+	AsyncBackingParams, Balance, ExecutorParams, SessionIndex, LEGACY_MIN_BACKING_VOTES,
+	ON_DEMAND_DEFAULT_QUEUE_MAX_SIZE,
+};
+use sp_runtime::Perbill;
 use sp_std::vec::Vec;
 
-use frame_support::traits::OnRuntimeUpgrade;
-
 use super::v10::V10HostConfiguration;
-type V11HostConfiguration<BlockNumber> = configuration::HostConfiguration<BlockNumber>;
+
+#[derive(Clone, Encode, PartialEq, Decode, Debug)]
+pub struct V11HostConfiguration<BlockNumber> {
```

Comment:
if nothing changed between v11 and v12, can we not simply do `pub type V11HostConfiguration = V12HostConfiguration`?

Reply:
Yes, that is what I've done initially but decided to make it a different struct to avoid any missuses as we have many unreleased migrations and another one  not merged here: https://github.com/paritytech/polkadot-sdk/pull/3181